src/integration/README.md

```diff
@@ -119,31 +161,65 @@ You might then think it makes sense to only write integration tests but there ar
 
-## What is DOM Testing Library?
-
-[DOM Testing Library](https://github.com/testing-library/dom-testing-library) provides testing utilities that encourage good testing practices. It provides methods to query the DOM for nodes in a way that's similar to how the user finds elements on the page. All queries available can be found [here](https://testing-library.com/docs/dom-testing-library/api-queries).
-
 ## What is a docblock pragma?
 
 A docblock pragma is a specially-formatted comment at the top of a test file. We can use docblock pragmas to alter the environment that tests run in.
 
-## What is the Given-When-Then stuff all about?
+## What is a snapshot?
 
-The Given-When-Then formula is a template intended to guide the writing of acceptance tests for a user story.
+This is a feature provided by Jest. It's not a snapshot of the graphical UI but a snapshot of a result that is typically returned from a function. This could even be HTML returned from DOM query selector e.g.
 
-- (Given) some context
-- (When) some action is carried out
-- (Then) a particular set of consequences can be observed
+```js
+it('should render the headline', () => {
+  expect(document.querySelector('h1').outerHTML).toMatchSnapshot();
+});
+```
 
-An example:
+The above would create a file in a colocated `__snapshots__` directory
 
-- Given I am on a Mundo article canonical page
-- When I am using the website
-- Then I can see an image with a caption.
+```
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`I can see the headline`] = `
+<h1 id="content"
+       "tabindex="-1"
+>
+  <span role="text">
+    <span lang="en-GB">
+      BBC News
+    </span>
+    ,
+    Pidgin
+    -
+    Home
+  </span>
+</h1>
+`;
+```
 
-## What is a snapshot?
+It's recommended not to snapshot large parts of the DOM because this creates brittle tests i.e. these tests would break often because the DOM changes whenever someone changes something such as a React component's structure and styles, translation configs and feature toggles. It is better to have smaller and more focused snapshots. More info on this approach can be found in this article [Effective Snapshot Testing](https://kentcdodds.com/blog/effective-snapshot-testing).
+
+An even better example of asserting a headline is on the page
+
+```js
+it('I can see the headline', () => {
+  const h1El = document.querySelector('h1');
 
-This is a feature provided by Jest. It's not a snapshot of the graphical UI but a snapshot of the underlying DOM tree that is used to catch unexpected changes.
+  expect(h1El).toBeInTheDocument(); // check the headline element is in the document
+  expect(h1El.getAttribute('id')).toBe('content'); // check for the id attribute
+  expect(h1El.getAttribute('tabindex')).toBe('-1'); // check for the tabindex attribute
+  expect(h1El).toBeTruthy(); // check there is some text inside the element
```

Comment:
Should this be `expect(h1El.textContent).toBeTruthy()` as per my previous comment?